.github/workflows/feature_deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Feature environment deployment
 
 on:
   push:
-    branches: [main-stbr-tfm-5]
+    branches: [feat/FN-1549/refactor-report-schedule]
```

Comment:
Reminder to change this. Do we want this as `main` now?

Reply:
yeah in a commit to fix the below may as well do main here for now